dbms/src/Flash/Mpp/MPPTunnelSetHelper.cpp

```diff
@@ -57,7 +82,6 @@ TrackedMppDataPacketPtr ToPacketV0(Blocks & blocks, const std::vector<tipb::Fiel
         tracked_packet->addChunk(codec_stream->getString());
         codec_stream->clear();
     }
-    blocks.clear();
```

Comment:
Why remove this?

Reply:
But `block.clear()` only clear the data in `block`, `blocks.size()` is not changed